in/in_command_test.go

```diff
@@ -109,25 +109,6 @@ var _ = Describe("In Command", func() {
 				Ω(versionID).Should(BeEmpty())
 			})
 
-			Context("when using a CloudFront domain", func() {
```

Comment:
No longer valid: the logic around `request.Source.CloudfrontURL` is now handled in `in/main.go`, not here. 

The `in_command` is passed an AWS Client. That client is configured in main with either a normal bucket and endpoint, or, if `cloudfront_url` is set, we split `cloudfront_url: https://d111111abcdef8.cloudfront.net` into `bucket: d111111abcdef8` and `endpoint: https://cloudfront.net`. `in_command` doesn't know whether it's talking to cloudfront or normal S3.